### ISSUE TITLE
Screen goes black after rotating

### ISSUE THREAD

**Issue by Packbat** [krille-chan/fluffychat#1285]
### Bug Description

When I rotate my phone from vertical to horizontal and back, the screen fades to black.

Not sure if related to #1275 but the steps to reproduce are the same.

### Steps to Reproduce

1. Open app in vertical orientation
2. Rotate from vertical to horizontal orientation
3. Rotate back

### Expected Behavior

Display should return to vertical format as normal.

### App Version

1.21.2

### Additional Platform Information

Samsung Galaxy A14 5G, Android 14, One UI 6.1

### Additional Context

Installed via FDroid.

**Comment by SteffoSpieler** [comment]
Seems like a duplication of #974

**Comment by bombermuecke** [comment]
> Seems like a duplication of #974

Yes. Fixing this bug seems to be a major problem, since there are many duplicates of it. In the meantime I have resigned myself and set up an automatism that deactivates the phone's auto-rotation function for this application.

**Comment by krille-chan** [comment]
Closing as duplicate of #979 (iOS rotation crash / RenderFractionalTranslation)
